src/main/proto/netflix/titus/agent.proto

```diff
@@ -371,3 +394,20 @@ message TaskIdentityStringDocument {
     // (Required) Signature produced by signing identity bytes.
     optional CertificateStringSignature signature = 2;
 }
+
+message TaskPodIdentityDocument {
+    // (Required) Serialized form of a TaskIdentity protobuf.
+    optional bytes podIdentity = 1;
+
+    // (Required) Signature produced by signing identity bytes.
+    optional CertificateSignature signature = 2;
+}
+
+// Version of TaskPodIdentityDocument that encodes all byte[]s as human-readable strings
```

Comment:
Not entirely clear to me why you want this, but it's fine if it's helpful.

Reply:
Yea I also wasn't sure. Maybe so one can just "curl" it? 
@rgulewich do you know?

If we do pod-as-bytes, this won't be very useful to anyone anyway.

Reply:
We use this for debugging, so inside the container you can use `curl`.  It's not strictly necessary, but it definitely helps track down issues.

Reply:
I dropped it in the most recent revision because the thing will only have bytes in it